src/js/components/CombinationCanvas.vue

```diff
@@ -157,19 +146,22 @@
                 return pt.matrixTransform(this.svgCanvas.getScreenCTM().inverse());
             },
         },
-        mounted() {
-            if (this.$route.params.scrollVersionID && this.$route.params.scrollID) {
-                this.artefacts = []
-                this.setScrollDimensions(this.$route.params.scrollID, this.$route.params.scrollVersionID)
-            }
-        },
         watch: {
             '$route' (to, from) {
-                if (to.params.scrollVersionID && to.params.scrollID
+                if (to.params.scrollVersionID !== '~' && to.params.scrollID !== '~'
                     && (to.params.scrollVersionID !== from.params.scrollVersionID 
                     || to.params.scrollID !== from.params.scrollID)) {
-                    this.artefacts = []
+                    this.scrollVersionID = to.params.scrollVersionID >>> 0
                     this.setScrollDimensions(to.params.scrollID, to.params.scrollVersionID)
```

Comment:
Seems strange that this takes two params that are the same to me.

Reply:
If you are referring to scrollID and scrollVersionID, they only look the same right now.  Scroll id designates a scholarly construct, like 4Q51, scroll version id points to a specific version of it (I might have 4 or 5 different versions of my reconstructed 4Q51).  Or were you referring to something else?